mttl/models/lightning/expert_module.py

```diff
@@ -144,6 +149,50 @@ def on_save_checkpoint(self, ckpt):
         ).expert_info.asdict()
 
 
+class SPLITExpertModule(ExpertModule):
+    """
+    Expert module used to train sparse mask with SPLIT mask updater.
```

Comment:
what is SPLIT mask updater ? Can we add a paper reference ?

Reply:
addressed

Reply:
addressed